geom/alg_relate.go

```diff
@@ -29,17 +29,18 @@ func Relate(a, b Geometry) (string, error) {
 		}
 
 		var flip bool
+		nonEmpty := b
 		if b.IsEmpty() {
-			b, a = a, b
+			nonEmpty = a
```

Comment:
This was to fix a lint error that complained that `a` was never used after this assignment. The lint error was accurate, and I think using a new `nonEmpty` variable better expresses what's going on here anyway.